pandas/core/generic.py

```diff
@@ -67,7 +67,8 @@
     args_transpose='axes to permute (int or label for object)',
     optional_by="""
         by : str or list of str
-            Name or list of names which refer to the axis items.""")
+            Name or list of names which refer to the axis items or index
+            levels.""")
```

Comment:
*Not blocking*: My OCD really wants you to not "orphan" the "levels" word at the end of this sentence :smile: .  However, that's largely a me problem, but if you do happen to come up with a wording that doesn't do this orphaning, by all means go for it.

Reply:
Heh, now it bothers me too! I reworked it to fit on one line. "Name or list of names matching axis items or index levels."